pkg/controllers/uiplugin/components.go

```diff
@@ -2,6 +2,10 @@ package uiplugin
 
 import (
 	"fmt"
+	"embed"
```

Comment:
the import list needs to be in alphabetical order or the linter will fail

Reply:
Done